TECHNOLOGY_CHOICES.md

```diff
@@ -132,3 +132,42 @@ https://github.com/Schematron/schematron-exist
 RiseClipse is also a good candidate, because it's dedicated on IEC 61850/CIM validation.
 Only thing is, it's not as flexible as using Schematron. But I really do like the combination Schematron and eXist-db/BaseX.
 
+# Microservice technology
+## Checks for determining microservice technology
+- Technology should be accepted by Alliander
+- 
+
+## Java Spring
+### Pros
+- Preferred language for Alliander
+- Lots of experience within CoMPAS community
+- Using Spring (Boot), it's very easy to quickly setup a microservice
+- Massive community
+
+### Cons
+- Pretty complex if you never worked with Java Spring
+
+## Go Micro
+### Pros
+
+### Cons
+- Not default accepted language for Alliander (decisive)
+
```

Comment:
Relative new; not a lot of libraries available and therefore requires more effort.